src/kits/tracker/SaveWindow.cpp

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
License: I think this code is mostly/all completely new, so you don't use the old OpenTracker license, but the modern Haiku license. Check the coding guidelines for more information.

Also in the header file.